Sources/Private/CoreAnimation/Animations/TransformAnimations.swift

```diff
@@ -255,6 +255,11 @@ extension CALayer {
           `rotation.y` values are not displayed correctly in snapshot tests
           """)
       }
+    } else {
+      // Due to issues with Samples/Issues/issue_1800.json, we can't enable X/Y rotation by default
+      try context.compatibilityAssert(
+        !containsXRotationValues && !containsYRotationValues,
+        "The Core Animation rendering engine doesn't fully-support X / Y rotation.")
     }
```

Comment:
We should do this even when running the snapshot tests. One of the test cases uses these assertions to track which sample json files are/aren't supported by the Core Animation rendering engine (in this case, `Test/__Snapshots__/AutomaticEngineTests/testAutomaticEngineDetection.Issues-issue_1800.txt`)

```suggestion
    }
    
    // Due to issues with Samples/Issues/issue_1800.json, we can't enable X/Y rotation by default
    try context.compatibilityAssert(
      !containsXRotationValues && !containsYRotationValues,
      "The Core Animation rendering engine doesn't fully-support X / Y rotation.")
```

Reply:
OK I see. Done it, just waiting for the UT to pass before committing